fieldservice_account/models/fsm_location.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class FSMLocation(models.Model):
+    _inherit = 'fsm.location'
+
+    @api.model
+    def get_default_customer(self):
+        if self.fsm_parent_id:
+            return self.fsm_parent_id.customer_id.id
+        return self.owner_id.id
+
+    customer_id = fields.Many2one(
+        'res.partner', string='Billed Customer', required=True,
```

Comment:
Why does the Customer have to be a required field?

Reply:
@dreispt Because the worker/system needs to know who is financially impacted by the service to contact him, to bill him, to get paid, etc...

CC: @wolfhall @patrickrwilson